hooks/node_handlers/base_export_handler.py

```diff
@@ -249,9 +245,7 @@ def repl_version(match):
                 version_path = paths.next()
                 fields = template.get_fields(version_path)
                 versions.add(int(fields["version"]))
-            versions = list(versions)
-            versions.sort()
-        return versions
+        return list(sorted(versions))
```

Comment:
actually found out the other day that sorted returns a list so we can forgo the list conversion here
```
>>> a =set([3,5,2,8])
>>> type(sorted(a))
<type 'list'>```

Reply:
this is because of python3 innit?

Reply:
```suggestion
        return sorted(versions)
```

Reply:
Yup, it's for Python 3, which will return an iterator.